server/grpc.go

```diff
@@ -34,17 +35,30 @@ func (s *GRPCServer) StreamAccessLogHandler(srv interface{}, ss grpc.ServerStrea
 	return err
 }
 
-func NewGRPCServer(opt Option) *GRPCServer {
+func NewGRPCServer(opt Option) (*GRPCServer, error) {
```

Comment:
__[golint]__
* [golint] exported function NewGRPCServer should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->